samples/algorithms/database-search/README.md

```diff
@@ -33,20 +38,19 @@ This sample walks through Grover's search algorithm. Oracles implementing the da
 
 This sample can be run in a number of different ways, depending on your preferred environment.
 
-### Python in Visual Studio Code or the Command Line
+### Visual Studio Code or the Command Line
 
-At a terminal, run the following command:
+At a terminal, run the following commands for each of the three examples:
 
 ```powershell
-python host.py
-```
+# Example 1:
+dotnet run simulate Microsoft.Quantum.Samples.DatabaseSearch.RunRandomSearch
 
```

Comment:
[nit] for usage on mybinder.org, splitting into multiple Markdown blocks allows for this Markdown file to be used as a command-line notebook to run each sample independently.
````suggestion
```

```
````